test/e2e/specs/sidebar-behaviour.test.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Internal dependencies
+ */
+import '../support/bootstrap';
+import { clearLocalStorage, newPost, newDesktopBrowserPage, setDesktopViewport, setMobileViewport } from '../support/utils';
+
+const SIDEBAR_SELECTOR = '.edit-post-sidebar';
+const ACTIVE_SIDEBAR_TAB_SELECTOR = '.edit-post-sidebar__panel-tab.is-active';
+const ACTIVE_SIDEBAR_BUTTON_TEXT = 'Document';
+
+describe( 'Publishing', () => {
+	beforeAll( async () => {
+		await newDesktopBrowserPage();
+	} );
+
+	afterEach( async () => {
+		await clearLocalStorage();
+		await page.goto( 'about:blank' );
+		await setDesktopViewport();
+	} );
+
+	it( 'Should have sidebar visible at the start with document sidebar active on desktop', async () => {
+		await newPost();
+		const { nodesCount, content, height, width } = await page.$$eval( ACTIVE_SIDEBAR_TAB_SELECTOR, ( nodes ) => {
+			const firstNode = nodes[ 0 ];
+			return {
+				nodesCount: nodes.length,
+				content: firstNode.innerText,
+				height: firstNode.offsetHeight,
+				width: firstNode.offsetWidth,
+			};
+		} );
+
+		// should have only one active sidebar tab.
+		expect( nodesCount ).toBe( 1 );
+
+		// the active sidebar tab should be document.
+		expect( content ).toBe( ACTIVE_SIDEBAR_BUTTON_TEXT );
+
+		// the active sidebar tab should be visible
+		expect( width ).toBeGreaterThan( 10 );
+		expect( height ).toBeGreaterThan( 10 );
+	} );
+
+	it( 'Should have the sidebar closed by default on mobile', async () => {
+		await setMobileViewport();
```

Comment:
Won't it give false positives because we are setting the desktop viewport first?

Reply:
We are setting the desktop viewport first as the default viewport but Gutenberg is not open. We are only opening Gutenberg in the `newPost` call.